### ISSUE TITLE
Question about  the ensemble target

### ISSUE THREAD

**Issue by HHHedo** [Goldesel23/Temporal-Ensembling-for-Semi-Supervised-Learning#8]
Hi, 
Thanks for your code. 
In your implement, the ensemble targets are network output before softmax. However, I think the target is the ensemble of output after softmax. Since in table 5 in the paper, output is the softmax.

**Comment by tensorfreitas** [comment]
Hi @HHHedo sorry for the late response. Only now I came back from vacations.

The last layer is a dense layer with softmax activation:

`wight_norm_layers.Dense.Dense(units=10, activation=tf.nn.softmax,`
